src/Bridge/Symfony/Bundle/Resources/config/data_provider.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+use ApiPlatform\Core\DataProvider\ChainCollectionDataProvider;
+use ApiPlatform\Core\DataProvider\ChainItemDataProvider;
+use ApiPlatform\Core\DataProvider\ChainSubresourceDataProvider;
+use ApiPlatform\Core\DataProvider\CollectionDataProviderInterface;
+use ApiPlatform\Core\DataProvider\ItemDataProviderInterface;
+use ApiPlatform\Core\DataProvider\Pagination;
+use ApiPlatform\Core\DataProvider\SubresourceDataProviderInterface;
+use Symfony\Component\DependencyInjection\ServiceLocator;
+
+return static function (ContainerConfigurator $container) {
+    $container->services()
+        ->set('api_platform.serializer_locator', ServiceLocator::class)
+            ->args([['serializer' => service('api_platform.serializer')]])
```

Comment:
You need to import that `service` function (see broken CI).
Symfony core does it by using `namespace Symfony\Component\DependencyInjection\Loader\Configurator` on top of each config file, but I'd rather add a `use function` statement  if it works without namespace.

Reply:
Change done, thank you :)